### PR TITLE
README.md updated

### PR DESCRIPTION
**Pull request by amit-metri**
README.md updated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,110 @@
-# BC Vaccine Card-Android
\ No newline at end of file
+# BCVAX-Android
```
Comment (pinakinkansara) [inline README.md:1]: Change to 
BC Vaccine Card Verifier-Android

File: README.md [hunk 2]
```diff
@@ -1 +1,110 @@
...
+- [Deployment](#markdown-header-deployment)
+  - [Versioning](#markdown-header-versioning)
+  - [Internal](#markdown-header-internal)
+  - [External](#markdown-header-external)
+- [Contributors](#markdown-header-contributors)
+
+## Project Resources
+---
+
+List any external project resources. Possible resources include:
```
Comment (pinakinkansara) [inline README.md:24]: This line is not required. Please remove it.

File: README.md [hunk 3]
```diff
@@ -1 +1,110 @@
...
+Architecture: __MVVM__
+
+	-  Architecture used for app: MVVM
+		- One activity with multiple fragments is used. Business logic is present in classes under utils package. View models associated with fragments seperate the UI from business logic. 
+		- Does it differ from standard architecture? No.
+		- Include UML diagram if needed. NA
+
+Dependency Injection: __Hilt__
+
+Concurrency: __Coroutines__
```
Comment (pinakinkansara) [inline README.md:42]: Add Kotlin Flow as well.

File: README.md [hunk 4]
```diff
@@ -1 +1,110 @@
...
+
+How is the app deployed and where?
+
+__Example:__ Builds generated manually, distributed through Google Play. CI/CD implementation is in progress
+
+### External
+
+How is the app deployed and where?
+
+__Example:__ Builds generated manually, distributed through Google Play.
```
Comment (pinakinkansara) [inline README.md:105]: It should be the explicit answer instead of an example.

File: README.md [hunk 5]
```diff
@@ -1 +1,110 @@
...
+
+### Versioning
+
+In version name 1.2, 1 stands for major change and 2 stands of minor change or patch fix.
+
+### Internal
+
+How is the app deployed and where?
+
+__Example:__ Builds generated manually, distributed through Google Play. CI/CD implementation is in progress
```
Comment (pinakinkansara) [inline README.md:99]: It should be the explicit answer instead of an example.

**Review by pinakinkansara — CHANGES_REQUESTED** [review]
Please make the requested changes.
Overall it looks good.

File: README.md (hunk 1 above)
Comment (amit-metri) [reply README.md]: changed

File: README.md (hunk 2 above)
Comment (amit-metri) [reply README.md]: changed

File: README.md (hunk 3 above)
Comment (amit-metri) [reply README.md]: changed

File: README.md (hunk 4 above)
Comment (amit-metri) [reply README.md]: changed

File: README.md (hunk 5 above)
Comment (amit-metri) [reply README.md]: changed

**Pull request merged**